src/main/webapp/puzzle.html

```diff
@@ -27,7 +27,10 @@ <h1 class="z-depth-2" id="puzzle-title">Puzzle</h1>
             <form id="answer-form" action="/user-level" method="POST" class="hidden">
                 <h3 class="enter-answer">Submit Your Answer Here:</h3>
                 <textarea name="answer" id="answer-input"></textarea>
-                <input class="submit-answer z-depth-1" type="submit" value="Submit Answer">
+                <input id = "submission" class="submit-answer z-depth-1" type="submit" value="Submit Answer">
+                <p id = "demo"></p>
+                <p id="error" hidden>Sorry that is incorrect. Try again!</p>
```

Comment:
is this necessary if we implement the html pages?

Reply:
no I need to delete that